smarts/core/trap_manager.py

```diff
@@ -145,6 +145,10 @@ def largest_vehicle_plane_dimension(vehicle):
                 ),
             )
             for v_id in sorted_vehicle_ids:
+                # Skip the capturing process if history traffic is used
+                if sim.scenario.traffic_history != {}:
```

Comment:
```suggestion
                if not sim.scenario.traffic_history:
```

Reply:
Thanks, though to be precise, the correction should be `if sim.scenario.traffic_history:`